### PR TITLE
fix(HMS-1847): Adding configmap to job

### PR DESCRIPTION
**Pull request by mayurilahane**
For adding config maps into a job:
1. have already created config map yaml in app-interface
2. then referenced it to the namespace in app-interface itself.
**3. Now have added its mount point in this MR's job file**

### CODE REVIEW

File: deploy/cloudwash_job.yaml [hunk 1]
```diff
@@ -31,6 +31,17 @@ objects:
...
                   - "--all"
+                volumeMounts:
+                  - name: config-volume
+                    mountPath: /opt/app-root/src/cloudwash/settings.yaml
+                    subPath: settings.yaml
+                volumes:
+                  - name: config-volume
+                    configMap:
+                      name: cloudwash-config
+                envFrom:
```
Comment (mshriver) [inline deploy/cloudwash_job.yaml:42]: This block shouldn't be necessary, the expected configmap is getting subPath mounted

Comment (mayurilahane) [reply deploy/cloudwash_job.yaml:40]: I removed the config map block only 
did you mean the whole volume block?

Comment (mshriver) [reply deploy/cloudwash_job.yaml:40]: just the `envFrom` map - but the `volumes` map is out of place too, its not part of Container spec, only `volumeMounts`.   `volumes` is part of the Pod spec, that map starts at L20.

File: deploy/cloudwash_job.yaml [hunk 2]
```diff
@@ -31,6 +31,14 @@ objects:
                   - "-d"
                   - "azure"
                   - "--all"
+                volumeMounts:
+                  - name: config-volume
+                    mountPath: /opt/app-root/src/cloudwash/settings.yaml
+                    subPath: settings.yaml
```
Comment (mshriver) [inline deploy/cloudwash_job.yaml:37]: suspect of this config with mountPath having the file, but I'm going to accept at this point to run in SRE cluster.

**Review by mshriver — APPROVED** [review]

**Pull request merged**